scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
I don't know really) I thought that you may know more about Lyapunov equation and can come up with some good definition. 

Here is a MATLAB's entry http://www.mathworks.com/help/control/ref/lyap.html Interesting that they don't require Q to be symmetric. How scipy works in this case?

As for definition, I would write something like this:

```
a : array_like, shape (M, M)
  Square matrix, corresponds to A in the equation above.  

q : array_like, shape (M, M)
  Square Hermitian matrix, corresponds to Q in the equation above.
```